pkg/spegel/spegel.go

```diff
@@ -232,29 +230,52 @@ func (c *Config) Start(ctx context.Context, nodeConfig *config.Node) error {
 		return err
 	}
 	mRouter.PathPrefix("/v2").Handler(regSvr.Handler)
-	mRouter.PathPrefix("/v1-" + version.Program + "/p2p").Handler(c.peerInfo())
+	mRouter.PathPrefix("/v1-{program}/p2p").Handler(c.peerInfo())
```

Comment:
Is this program value inject internally in Spegel now?

Reply:
This is just our route to the handler that returns the address list that is used by our custom agent bootstrapper. I missed this one when switching to `{program}` in https://github.com/k3s-io/k3s/pull/11471